libweasyl/libweasyl/models/tables.py

```diff
@@ -219,8 +219,7 @@ def default_fkey(*args, **kwargs):
     Column('settings', CharSettingsColumn({
         'p': 'pending',
     }, length=20), nullable=False, server_default='p'),
-    Column('unixtime', WeasylTimestampColumn(), nullable=False,
-           server_default=text(u"(date_part('epoch'::text, now()) - (18000)::double precision)")),
+    Column('created_at', DateTime(timezone=True), nullable=False, server_default=func.now()),
```

Comment:
Can you use `TIMESTAMP` instead? It’s not like we’re going to stop using PostgreSQL, but [the documentation][1] does say:

> It is recommended to make use of the `TIMESTAMP` datatype directly when using this flag

(but the PostgreSQL one that’s already imported, not the one it links to.)

I don’t know if this changes the generated migration.


[1]: https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.DateTime

Reply:
Updated, IIRC, they all end up as the same PostgreSQL type, tested it locally, and everything works fine when switched to TIMESTAMP instead of DateTime